docs/reference/modules/gateway.asciidoc

```diff
@@ -48,3 +48,12 @@ as long as the following conditions are met:
     Recover as long as this many data nodes have joined the cluster.
 
 NOTE: These settings only take effect on a full cluster restart.
+
+=== Dangling indices
+
+When a node joins the cluster, any shards stored in its local data directory 
+directory which do not already exist in the cluster will be imported into the 
+cluster. This functionality is intended as a best effort to help users who 
+lost all master nodes. If a new master node is started which is unaware of 
+the other indices in the cluster, adding the old nodes will cause the old 
+indices to be imported, instead of being deleted.
```

Comment:
There's an extra "directory" in the first sentence. 

I recommend avoiding future tense unless it's really necessary--and in this case I think it would be clearer to stick to present tense. I'd make the following tweaks:

When a node joins the cluster, any shards stored in its local data directory 
that do not exist in the cluster are imported into the cluster. This functionality
is intended as a best effort to help you recover if a cluster loses all master
nodes. If a new master node is started that is unaware of the other indices in
the cluster, the old indices are imported as the old nodes join the cluster, not
deleted.

I reflexively use "that" instead of "which" for dependent clauses--though that "rule" isn't as hard and fast as some folks like to think.

We normally just keep pushing updates to doc PRs until it's ready to merge, then squash to a single commit and merge. Occasionally we'll keep multiple commits when merging, particularly if there are changes that are related, but not reflected in original commit messages.

LGTM as long as the directory stutter is fixed.